From c5a691c13a2f589c2f72f16edc2817f29639e77a Mon Sep 17 00:00:00 2001
From: Thomas Parrott <thomas.parrott@canonical.com>
Date: Thu, 21 Sep 2023 22:12:19 +0100
Subject: [PATCH] shared/simplestreams/products: Fix regression in
 lxd_combined.tar.gz handling

For reasons that are likely lost to history simplestreams uses lxd.tar.xz
whereas the combined file is lxd_combined.tar.gz (not xz).

Signed-off-by: Thomas Parrott <thomas.parrott@canonical.com>
---
 shared/simplestreams/products.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/shared/simplestreams/products.go b/shared/simplestreams/products.go
index ef7e1e45c1c4..a07e4d5b1b5b 100644
--- a/shared/simplestreams/products.go
+++ b/shared/simplestreams/products.go
@@ -10,7 +10,7 @@ import (
 	"github.com/canonical/lxd/shared/osarch"
 )
 
-var lxdCompatCombinedItems = []string{"lxd_combined.tar.xz", "incus_combined.tar.xz"}
+var lxdCompatCombinedItems = []string{"lxd_combined.tar.gz", "incus_combined.tar.gz"}
 var lxdCompatItems = []string{"lxd.tar.xz", "incus.tar.xz"}
 
 // Products represents the base of download.json.
